database/migrations/2022_07_08_085310_add_allow_performance_to_osu_builds.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddAllowPerformanceToOsuBuilds extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('osu_builds', function (Blueprint $table) {
+            $table->boolean('allow_performance')->default(false);
```

Comment:
```suggestion
            $table->boolean('allow_performance')->default(false)->after('allow_bancho');
```

Just for my own sanity.

Also as discussed, let's add an index on this column.